multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -56,6 +69,35 @@ coreos:
         [Install]
         WantedBy=multi-user.target
 
+{{ if eq .ContainerRuntime "rkt" }}
+    - name: rkt-api.service
+      enable: true
+      content: |
+        [Service]
+        ExecStart=/usr/bin/rkt api-service
+        Restart=always
+        RestartSec=10
+        [Install]
+        RequiredBy=kubelet.service
+        Before=kubelet.service
```

Comment:
Oops sorry @pbx0, `Before` and `After` both go in the `[Unit]` section, not `[Install]`